Header header
bool safe
bool is_flat

float32 pad_force

uint8 rivet_count

geometry_msgs/Point[] rivet_pos

